examples/build_model.ipynb

```diff
@@ -176,6 +139,14 @@
     "### Setup complete Wflow model "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "a5062c03",
+   "metadata": {},
+   "source": [
+    "Once we have prepared our data catalog and model build settings file, we are ready to ask HydroMT to build our Wflow model. This can be done from the command line and here is an example for the Northern part of the Piave basin in Italy:"
+   ]
```

Comment:
Small comment:
I'd split the second sentence into two parts, e.g:
`Building the model can be done from the command line. Down below is an example for the Northern part of the Piave basin in Italy:` 

You can disregard it if you disagree.